sdk/resourcemanager/azure-resourcemanager-samples/pom.xml

```diff
@@ -103,7 +103,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
-      <version>4.3.0</version> <!-- {x-version-update;com.azure:azure-cosmos;dependency} -->
+      <version>4.3.1</version> <!-- {x-version-update;com.azure:azure-cosmos;dependency} -->
```

Comment:
@JimSuplizio why do we have release version here instead of 4.4.0-beta.1, is it because this is sample